pkg/imgpkg/cmd/push.go

```diff
@@ -148,8 +197,52 @@ func (po *PushOptions) pushImage(registry registry.Registry) (string, error) {
 		return "", fmt.Errorf("Images cannot be pushed with '.imgpkg' directories, consider using --bundle (-b) option")
 	}
 
+	baseImageName, err := po.stripTag()
+	if err != nil {
+		return "", err
+	}
+
+	baseRef, err := regname.NewTag(po.ImageFlags.Image, regname.WeakValidation)
+	if err != nil {
+		return "", fmt.Errorf("Parsing '%s': %s", po.BundleFlags.Bundle, err)
+	}
+
+	// Append the base image_tag to the list of refs to upload
+	uploadRefs = append(uploadRefs, baseRef)
```

Comment:
I think that this might be more readable if we do:

```suggestion
	imageRefs := []string{baseRef}
```